docs/docs/dashboard.md

```diff
@@ -15,16 +15,38 @@ Furthermore, the dashboard serves as a gateway to various essential sections, su
 
 ## Workspace Manager
 
-The workspace manager is located on the top left corner of the dashboard. Clicking on the workspace manager will open a dropdown menu with a list of all the workspaces you are a part of. You can switch between workspaces by clicking on the workspace name from the dropdown menu.
-
-You can also create a new workspace by clicking on the `Add new workspace` button on the bottom of the dropdown menu. Clicking on this button will open a modal, enter the name of the workspace and click on the `Create Workspace` button to create a new workspace. Workspaces can be **renamed** by clicking on the `edit` icon on the right side of the workspace name.
+The workspace manager is located on the bottom left corner of the dashboard. Clicking on the workspace manager will open a dropdown menu with a list of all the workspaces you are a part of. You can switch between workspaces by clicking on the workspace name from the dropdown menu.
 
 <div style={{textAlign: 'center'}}>
 
 <img className="screenshot-full" src="/img/dashboard/workspacemenu.png" alt="Dashboard"/>
 
 </div>
 
+### Add new workspace
+
+On clicking the `Add new workspace` button, a modal will open where you can enter the name of the workspace, enter the unique workspace slug, and can see the preview of the workspace URL. Click on the `+ Create Workspace` button to create a new workspace.
+
+Previously, the workspace slug was the workspace id which was generated automatically. An example of the workspace slug was `https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0`. The new workspace slug is a unique name that you can choose for your workspace. An example of the new workspace slug is  `https://tooljet.com/apac-team `.
```

Comment:
Previously, the workspace slug was an automatically generated workspace ID, like this: https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0'. Now, you can choose a custom, unique slug for your workspace (can add a ss here for reference) which will generate a more accessible or readable URL such as https://tooljet.com/apac-team."

Reply:
also, please check if we can add a screenshot here - it will be good. if we are already adding here, then it will be good to highlight the specific areas / parts so that user gets it.

Reply:
```suggestion
Previously, the workspace slug was an automatically generated workspace ID, like this: `https://tooljet.com/262750db-b2b8-4abb-9404-8995c2ecb2a0`. Now, you can set a custom, unique slug for your workspace which will generate a more accessible or readable URL such as `https://tooljet.com/apac-team)`.
```

Reply:
Made the changes and the screenshot is already there.